readme.md

```diff
@@ -3,7 +3,7 @@
 Protractor-sync builds on protractor and provides:
 
 * Synchronous-style test writing (using fibers, behind the scenes)
-* Polling mechanisms for testing asynchronous apps (polledExpect, elementFinder.waitUntil & .waitUntilRemoved, browser.waitFor)
+* Polling mechanisms for testing asynchronous apps (polledExpect, elementFinderSync.waitUntil & .waitUntilRemoved, browserSync.waitFor)
```

Comment:
waitFor is no longer a method on browserSync. It's a standalone method that lives in utility.ts.